app/graphql/concerns/validate_student_submission.rb

```diff
@@ -39,9 +39,10 @@ def validate(_object, context, value)
 
   class AttemptedMinimumQuestions < GraphQL::Schema::Validator
     def validate(_object, _context, value)
-      target = Target.find_by(id: value[:target_id])
+      assignment =
```

Comment:
Shouldn't the mutation be accepting `assignment_id` instead of `target_id` 🤔

Reply:
@bodhish A timeline_event/submission still belongs_to a target and not assignment. So the `CreateSubmissionMutation` is taking in a target_id.